osgar/drivers/sicklidar.py

```diff
@@ -37,7 +39,26 @@ def parse_raw_data(raw_data):
         scan_start = 26
         scan_end = scan_start + scan_size
         dist = [int(x, 16) for x in data[scan_start:scan_end]]
-        return dist
+
+        remission = None
+        if scan_end + 100 < len(data):  # TODO properly detect presence RSSI
```

Comment:
So, what is the proper way to detect presence of RSSI?

Reply:
None means that it is not enabled, and [] that it is enabled but we did not get any data ... actually the workaround with empty scan seems to be related to broken(?) sensor, cause unknown. And if you do not get any data it is not clear if RSSI is  or is not enabled, so I would guess that both outputs are acceptable.

Reply:
In this comment I was actually asking about the "TODO".

Reply:
I believe that there should be some bit-flag what sections are and are not present in the data. I would review this again with integration of TIM311 (UBS lidar) and LMS-100.